fenix/nimbus.fml.yaml

```diff
@@ -138,6 +138,21 @@ features:
       - channel: nightly
         value:
           enabled: false
+  
+  client-deduplication:
+    description: A feature to control the sending of the client-deduplication ping.
+    variables:
+      enabled:
+        description: If true, the ping will be sent on a similar schedule as the baseline ping.
+        type: Boolean
+        default: false
+    defaults:
+      - channel: nightly
+        value:
+          enabled: true
```

Comment:
This will default on for all nightly builds, I think you want it to be false in all cases.